IdentityCore/src/pop_manager/MSIDDevicePopManager.m

```diff
@@ -84,7 +84,7 @@ - (NSDictionary *)buildPayloadDict:(NSString *)accessToken
     }};
     [payload addEntriesFromDictionary:cnf];
     
-    [payload setObject:[NSString stringWithFormat:@"%lu", (long)[[NSDate date] timeIntervalSince1970]] forKey:@"ts"];
+    [payload setObject:[NSNumber numberWithLong:(long)[[NSDate date] timeIntervalSince1970]] forKey:@"ts"];
```

Comment:
nit: Use [NSNumber numberWithUnsignedLong:]

Reply:
Thanks @ameyapat for the suggestion.